assembly/src/etc/tornado.properties

```diff
@@ -1,8 +1,35 @@
 tornado.opencl.print = false
 tornado.debug = false
-tornado.platform = 0
+#
+# Index of the default driver (OpenCL vs PTX) used for TaskSchedule.
+# Please check output of "tornado --devices" for the correct value.
+tornado.driver = 0
+#
+# Index of the default default device within default driver 
+# used for TaskSchedule.
+# Please check output of "tornado --devices" for the correct value.
 tornado.device = 0
 tornado.events.log = false
 tornado.vectors.enable = true
 tornado.kernels.parallelise = true
 tornado.opencl.gpu.block.x = 512
+#
+# Index of the default platform (like AMD, Intel, NVIDIA etc)
+# used in OpenCL-specific applications.
+# Please note that this property does not affect default driver/device
+# settings for TaskSchedule.
+tornado.opencl.platform = 0
+#
+# Index of the default device withing default platform
```

Comment:
Index of the default OpenCL device within a selected platform.
This index corresponds to the OpenCL device level that the Operating System sees within an OpenCL platform.
Note that this property does not affect the default `driver:device` setting for the TaskSchedule.